manifests/params.pp

```diff
@@ -161,7 +161,7 @@
   $puppetca_cmd       = $puppet::params::puppetca_cmd
   $puppet_group       = 'puppet'
   $puppetdir          = $puppet::params::dir
-  $autosign_location  = "${puppetdir}/autosign.conf"
+  $autosign_location  = $puppet::params::autosign
```

Comment:
I thought it was used in the pupeptca.yml template, but investigation shows that puppetdir/autosign.conf is used in the smart-proxy code...